DATABASE.md

```diff
@@ -20,3 +20,9 @@ This version defaults to starting postgres server version 11. To change to versi
 ### Clear the database
 
 `pslq -f db/truncate-tables jadedb jade`
+
+
+### Gradle commands
+
+To run database updates (this is not done automatically), run `./gradlew update`
```

Comment:
Nice!

Reply:
Is there a way to rollback an update or only update to a specific point?